src/bin/utils/options.ts

```diff
@@ -118,6 +125,21 @@ const rootKeyPairBits = new commander.Option(
   'Bit size of root key pair',
 ).argParser(binParsers.parseNumber);
 
+const seedNodes = new commander.Option(
+  '-sn, --seed-nodes [nodeId1@host:port;nodeId2@host:port;...]',
+  'Seed node address mappings',
+)
+  .argParser(binParsers.parseSeedNodes)
```

Comment:
Then subsequently both can be combined together.

```
pk --seed-nodes='<default>' --network=testnet
```

So `parseSeedNodes` can return a type that looks like this:

```
[
  {
    nodeId: { host, port }
  }, 
  true/false
]
```

During processing:

```
const [seedNodes, defaultSeedNodes] = options.seedNodes;
if (defaultSeedNodes) {
  Object.assign(seedNodes, options.network);
}
```

Reply:
The element of the tuple is `NodeMapping`.